docs/src/main/sphinx/release/release-467.md

```diff
@@ -0,0 +1,105 @@
+# Release 467 (4 Dec 2024)
+
+## General
+
+* Change default protocol for internal communication to HTTP/1.1. ({issue}`24281`)
+* Fix query failures or wrong output in `SHOW STATS` when a connector returns
+  `NaN` values for table statistics. ({issue}`24315`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+* Reduce image size and remove package manager.  ({issue}`24281`)
+
+## CLI
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+## Druid connector
+
+## Elasticsearch connector
+
+## Exasol connector
+
+## Faker connector
+
+## Google Sheets connector
+
+## Hive connector
+
+## Hudi connector
+
+## Iceberg connector
+
+* Add the `all_manifests` metadata table. ({issue}`24330`)
```

Comment:
We got no docs for that .. @ebyhr and @Ordinant are working towards it I think

Reply:
https://github.com/trinodb/trino/issues/24339

Reply:
I know .. I pinged @Ordinant already to see who can help